guides/_subsections/store-credentials.md

```diff
@@ -0,0 +1,47 @@
+## [Store Credentials](#store-credentials)
+If you have Java Management Extensions (JMX) or HTTP authentication enabled on your containerized JVMs, Cryostat will prompt you to enter your credentials before it can access the JDK flight recordings on your target JVMs. You can <a href="#configure-credentials-storage">Configure Credentials Storage</a> and choose whether these credentials are held in browser memory for the current session only, or if they are uploaded to Cryostat's credentials keyring.
+
+If you intend to use Cryostat <a href="#create-an-automated-rule">Automated Rules</a>, then you should store target application credentials in Cryostat's credentials keyring, which is outlined below. Even if you do not use Automated Rules, you may find it more convenient to store credentials in the keyring. This way, the Cryostat web UI does not need to prompt you for credentials when you manually access JFR information about target applications.
+
+<ol>
+  <li>
+    {% include howto_step.html
+      summary="Navigate to the Security tab"
+      image-name="2.3.0/navigate-to-security.png"
+      text="
+        First, navigate to the <i>Security</i> tab. The Store Credentials table lists all credentials in Cryostat's keyring. Click <i>Add</i> to define a new set of credentials.
```

Comment:
We are defining a single credential on the modal right? Or does `a new set of credentials` means something else?

Reply:
Good question. I tend to say "credentials" in the plural even when talking about a singular definition, but I'm not really sure why. Maybe because the username and password together form the "credentials"? I'm fine with changing this.

Reply:
Make sense! Instead of `a new set of credentials`, how about just `new credentials`?